lib/regions.js

```diff
@@ -73,22 +73,54 @@ function buildPageTemplate(templates, page) {
     return pageTemplate;
 }
 
+/**
+ * Returns a boolean if there are two regions with the same name
+ * @param {Object} regions
+ * @param {Object} regionData
+ * @returns {boolean}
+ */
+function checkDuplicateRegionName(regions, regionData) {
+    let result = false;
+    Array.from(regions).forEach((region) => {
+        if (region.name === regionData.name) {
+            result = true;
+        }
+    });
+
+    return result;
+}
+
 /**
  * Parse a template content and return an array of unique region objects
  * @param {string} content
  * @returns {Array}
  */
 function parseRegions(content) {
-    const regionRegex = /{{3}\s*region\s+.*?name=["|']([\w-]+)["|'].*?}{3}/g;
+    const regionRegex = /{{3}\s*region\s+.*?name=["|']([\w-]+)["|'].*?}{3}?/g;
+    const translationRegex = /{{3}.*?translation=["|'](i18n.RegionName.([\w-])+)["|'].*?}{3}/g;
```

Comment:
We should check for `\s` before `translation` similar to the regionRegex. Can you check if this will pass for `xyztranslation`?

Reply:
oh yeah, good call out, I will update now. Yeah this will still pass even with we have `{{{region xyztranslation="i18n.RegionName.blahblah"}}}`